tests/ludwig/modules/test_embedding_modules.py

```diff
@@ -19,7 +19,7 @@ def test_embed(
         embedding_size=embedding_size,
         representation=representation,
     )
-    inputs = torch.randint(0, 2, size=(2, 1)).bool()
+    inputs = torch.randint(0, 2, size=(2, 1)).bool().to('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Could you create a global `DEVICE` variable, and use that while moving all `inputs` to the correct device?